source/fundamentals/authentication/mechanisms.txt

```diff
@@ -201,6 +181,49 @@ The driver checks for your credentials in the following sources in order:
       .. literalinclude:: /code-snippets/authentication/aws.js
          :language: javascript
 
+   .. tab:: Environment Variables
+      :tabid: environment variables
+
+      To authenticate to your MongoDB instance using AWS credentials stored in
+      environment variables, set the following variables by using
+      a shell as shown in the following example:
```

Comment:
Small non-blocking suggestion to avoid repeated phrase "the following"

```suggestion
      To authenticate to your MongoDB instance using AWS credentials stored in
      environment variables, set the following variables by using
      a shell:
```